module/Olcs/config/module.config.php

```diff
@@ -872,6 +878,21 @@
             RouteParam\LicenceFurniture::class,
             RouteParam\Licence::class,
         ],
+        LicenceEnableDisableMessagingController::class => [
+            RouteParam\Conversation::class,
```

Comment:
Can we do without Conversation here?

Reply:
What do you mean?

Reply:
We'd have to change the path to something organisation based - probably out of scope at this point